### PR TITLE
Simple fixes

### PR DESCRIPTION
**Pull request by ockham**
Two simple bug fixes.

### CODE REVIEW

**Comment by jancborchardt** [comment]
Please review this @babelouest @Garfonso @ben-denham

**Comment by jancborchardt** [comment]
@ockham thank you for the contribution! Please also review some other pending pull requests in exchange. :)

**Comment by jbtbnl** [comment]
:+1:

**Comment by ockham** [comment]
@DeepDiver1975 review pls

File: lib/controller/groupcontroller.php [hunk 1]
```diff
@@ -39,7 +39,7 @@ public function getGroups() {
 				$ids = $this->tagMgr->getIdsForTag($tag['name']);
 				$tag['contacts'] = $ids;
 			} catch(\Exception $e) {
-				$this->api->log(__METHOD__ . ' ' . $e->getMessage());
+				\OCP\Util::writeLog('contacts', __METHOD__ . ' ' . $e->getMessage(), \OCP\Util::ERROR);
```
Comment (vgezer) [inline lib/controller/groupcontroller.php]: Not sure if a big deal, but I would put a comma next to method for consistency with the other `writeLog` usages.

```
\OCP\Util::writeLog('contacts', __METHOD__.', ' . $e->getMessage(), \OCP\Util::ERROR);
```

**Comment by vgezer** [comment]
Just one minor issue, otherwise :+1:

**Comment by vgezer** [comment]
Can you squash commits please? So that we can merge this:

https://help.github.com/articles/about-git-rebase#rebasing-commits-against-a-point-in-time

**Comment by ockham** [comment]
@wakeup You mean sqashing all 5? They aren't really that much related. Is that really necessary for merging them?

**Comment by ockham** [comment]
As per @wakeup's suggestion, I've squashed the comma fix (5eef574) into the other (67228e3) and merged into master.

**Comment by vgezer** [comment]
@ockham Great. I also read in the IRC channel saying that we do not need to squash the commits :).

**Pull request closed without merging**